README.md

```diff
@@ -99,6 +99,16 @@ The prompt shows information you need while you're working, while staying sleek
    $ cargo install starship
    ```
 
+   #### Arch Linux (AUR)
+
+   Starship is available on the AUR under the package name `starship`. Install it with
```

Comment:
I think having this on one line may look nicer.

```suggestion
   Starship is available on the AUR under the package name `starship`. Install it with `yay` or your favorite helper.
```

Reply:
Agreed.